src/Akeneo/Pim/Enrichment/Bundle/Command/MigrateToUuid/MigrateToUuidCommand.php

```diff
@@ -85,17 +108,65 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                     \sprintf('Step done in %0.2f seconds', $step->getDuration()),
                     $logContext->toArray(['migration_duration_in_second' => time() - $startMigrationTime])
                 );
-            } else {
-                $step->setStatusSkipped();
-                $this->logger->notice(
-                    'No items to migrate. Step skipped.',
-                    $logContext->toArray(['migration_duration_in_second' => time() - $startMigrationTime])
-                );
             }
-        }
 
-        $this->logger->notice('Migration done!', ['migration_duration_in_second' => time() - $startMigrationTime]);
+            $this->logger->notice('Migration done!', ['migration_duration_in_second' => time() - $startMigrationTime]);
+        } finally {
+            $this->stop();
+        }
 
         return Command::SUCCESS;
     }
+
+    private function start()
+    {
+        $this->connection->executeQuery(<<<SQL
+            INSERT INTO `pim_one_time_task` (`code`, `status`, `start_time`, `values`) 
+            VALUES (:code, :status, NOW(), :values)
```

Comment:
```suggestion
            VALUES (:code, :status, NOW(), :values)
            ON DUPLICATE KEY UPDATE status = 'started', start_time=NOW();
```

In case of the pod is hard kill. Otherwise the command will always fail here

Reply:
Already fix :partying_face: